### ISSUE TITLE
Translate not implemented for VoxelGrid geometry

### ISSUE THREAD

**Issue by mark-searle99** [intel-isl/Open3D#3361]
**Describe the bug**
When applying the translate function to a VoxelGrid geometry I am getting the following error:
"RuntimeError: [Open3D ERROR] Not implemented"

**To Reproduce**
Run translate on VoxelGrid geometry. Geometry created using 'create_from_point_cloud()' function.

- Operating system: Windows 10 64-bit
- Python version: Python 3.8
- Open3D version: 0.12.0
- Is this remote workstation?: no
- How did you install Open3D?: pip

**Additional context**

**Comment by yogeshhk** [comment]
I can take it up if this needs to be implemented. Being a newbie, it will help if existing contributors can give some hints, where to look and implement. I will start digging in anyways.

**Comment by zhengminxu** [comment]
I am also trying to use the translate and rotate functions for VoxelGrid. Would really appreciate it if you could implement both!

**Comment by elevenjiang1** [comment]
me too,hoping can use this function!

**Comment by chunibyo-wly** [comment]
Hello, I want to implement these 4 functions.
Could you please give me some hints to achieve this? @yxlao 

1. In translate function, it can directly apply translate to voxel grid `origin_`
2. In scale function, apply scale may cause voxel cube prism lengths become unequal. (Maybe force scale parameter be equal? And directly scale voxel size)
3. In rotate function, it confused me that should voxel coordinate should be rotated too? The problem is other functions are calculated by default using a coordinate system parallel to the world coordinate axis.

Or can you please give me some reference?

**Comment by elevenjiang1** [comment]
你好,我是蒋俊南,我已收到你的邮件,谢谢

**Comment by elevenjiang1** [comment]
> me too,hoping can use this function!

Yes, now I solve my request by change the origin of voxel, seems that it can solve problem~
